README.adoc

```diff
@@ -108,23 +108,31 @@ yarn build && yarn start [-p 29703]
 
 === Database
 
-To prepare the `users` table you'll need to run:
-[source, shell]
-----
-yarn db:create-users-table
-----
+We made the choice to use https://aws.amazon.com/dynamodb/[DynamoDB].
 
-If you want to reset the `users` table, run:
-[source, shell]
-----
-yarn db:reset-users-table
-----
+The `users` DynamoDB table is created automatically during the docker compose launch.
 
-If you want to delete the `users` table, run:
-[source, shell]
-----
-yarn db:delete-users-table
-----
+If you want to interact with the database, you can use the https://docs.aws.amazon.com/cli/latest/reference/dynamodb/[AWS CLI], of use one of the following npm scripts that use the SDK:
+
+
+|===
+| Name | Description
+
+| `db:create-users-table`
```

Comment:
How should we run those command, still with `yarn <cmd>`?

Reply:
Yes, as they are described as `npm scripts`, even if we don't really need them any more :).

Reply:
👍